apps/frontend/src/components/cards/controltable/ControlTable.vue

```diff
@@ -57,11 +57,21 @@
         </template>
 
         <template #id>
-          <ColumnHeader
-            text="ID"
-            :sort="sortId"
-            @input="set_sort('id', $event)"
-          />
+          <v-row class="pa-3">
+            <ColumnHeader
+              text="ID"
+              :sort="sortId"
+              @input="set_sort('id', $event)"
+            />
+            <v-tooltip bottom>
+              <template #activator="{on, attrs}">
+                <v-icon style="cursor: pointer" v-bind="attrs" x-small v-on="on"
+                  >mdi-help</v-icon
+                >
+              </template>
+              <span>ID <br />(Legacy ID) </span>
```

Comment:
@HackerShark should add some additional text here to help explain the concept of legacy ids, but don't let that block this pr's progress.  We can make a new pr with that additional text if necessary.

Reply:
Like we discussed, people who've been using older STIGs will understand. Others won't care.